index.js

```diff
@@ -4,6 +4,30 @@ function restartGame(){
 
 // Wrapping entire code in anonymous function and calling it, so that user doesn't have access to cardImageSrcs
 (() => {
+    const game = document.querySelector('.memory-game');
+  function openM(){
+    alert("man");
+  }

```

Comment:
why is this used

Reply:
> why is this used

this was not coded by me but i have removed it